### DISCUSSION TITLE
A way to use ESM instead of cjs

### DISCUSSION THREAD

**Discussion by itsRVX** [sequelize/sequelize#16036] [Help & Questions] [answered]
I cant seem anything on this. I'm using ESM with javascript* not typescript. and using the CLI creates cjs files and its going to be very annoying redoing each model/migrations. Is there an easy way to tell sequelize im using ESM and not cjs?

**Comment by ephys** [comment, marked as answer]
The main library supports ESM, but the CLI does not
Support for the CLI will be implemented as part of a complete rewrite we'll work as part of Sequelize 7

**Reply by itsRVX** [reply]
Is there any time frame on when v7 will be released?

**Reply by ephys** [reply]
Please see https://github.com/sequelize/sequelize/discussions/15736
